screwdriver.yaml

```diff
@@ -0,0 +1,100 @@
+shared:
+    image: node:12
+
+jobs:
+    main:
+        annotations:
+            screwdriver.cd/ram: TURBO
+            screwdriver.cd/cpu: TURBO
+        requires:
+            - ~pr
+            - ~commit
+            - ~sd@73:publish #artifact-bookend https://cd.screwdriver.cd/pipelines/73
+        steps:
+            - setup-ci: git clone https://github.com/screwdriver-cd/toolbox.git ci
+            - install: npm install
+            - duplicate: NPM_FILTER=screwdriver- ./ci/npm-dups.sh
+            # - test: npm test
```

Comment:
we should add these back and let npm handle it.

Reply:
Yes, adding the tests and test scripts